docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -294,6 +352,41 @@ You can render the ``_id`` field to readable values with a cast to ``VARCHAR``:
      55b151633864d6438c61a9ce  |        1 | bad         |       50.0 | 2015-07-23
     (1 row)
 
+ObjectId Timestamp Utility Functions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In MongoDB the first four bytes of each `ObjectId <https://docs.mongodb.com/manual/reference/method/ObjectId>`_ contain
+an embedded timestamp of its creation time. Trino provides a couple of functions to take advantage of this MongoDB feature.
+
+``objectid_timestamp``
+^^^^^^^^^^^^^^^^^^^^^^
+
+Allows you to extract the timestamp from a given ObjectId. It can be used as follows:
+
+.. code-block:: sql
+
+    SELECT '_id' as ID, objectid_timestamp('_id') as TS
+    FROM collection;
+
+
+``timestamp_objectid``
+^^^^^^^^^^^^^^^^^^^^^^
+
+Creates an ObjectId from a given timestamp. In MongoDB you can filter all the documents created after August 7th, 2021
+with a query like this:
+
+.. code-block:: text
+
+    db.collection.find({"_id": {"$gt": ObjectId("610ec8280000000000000000")}})
+
+In Trino, the same can be achieved with this query:
+
+.. code-block:: sql
+
+    SELECT *
+    FROM collection
+    WHERE '_id' > timestamp_objectid(TIMESTAMP '2021-08-07 17:51:36 +00:00');
```

Comment:
Single quotation expects a literal, not a column. 
```suggestion
    WHERE  _id > timestamp_objectid(TIMESTAMP '2021-08-07 17:51:36 +00:00');
```

Reply:
removed quotes